### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
First Commit

### CODE REVIEW

**Review by cs-shravan-konthalapally — APPROVED** [review]

File: riskiq-whoisiq/info.json [hunk 1]
```diff
@@ -0,0 +1,861 @@
...
+        {
+          "title": "Maximum Results",
+          "name": "maxResults",
+          "tooltip": "Maximum number of results to return. Defaults to 1000 and maximum value is 10000..",
+          "description": "Maximum number of results to return. Defaults to 1000 and maximum value is 10000..",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "validation": {
```
Comment (cs-prerna-joshi) [inline riskiq-whoisiq/info.json:624]: when we enter teh jinja expression the validation will fail. So please remove this. Handle the validation in the code.

File: riskiq-whoisiq/info.json [hunk 2]
```diff
@@ -0,0 +1,861 @@
...
+      "enabled": true
+    },
+    {
+      "operation": "get_phone",
+      "title": "Get Phone Number",
+      "description": "Get WHOIS records associated with a phone number.",
+      "category": "investigation",
+      "annotation": "get_phone",
+      "output_schema": {
+        "results": 0,
```
Comment (cs-prerna-joshi) [inline riskiq-whoisiq/info.json]:
```suggestion
        "results": "",
```

File: riskiq-whoisiq/info.json [hunk 3]
```diff
@@ -0,0 +1,861 @@
...
+        },
+        {
+          "title": "Exact Match",
+          "name": "exact",
+          "tooltip": "Search for an exact match. Valid values are true and false.",
+          "description": "Search for an exact match. Valid values are true and false.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "checkbox"
```
Comment (cs-prerna-joshi) [inline riskiq-whoisiq/info.json]: set the default value

File: riskiq-whoisiq/info.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply riskiq-whoisiq/info.json:624]: I have changed the type to integer and validation is working as expected

Comment (cs-prerna-joshi) [reply riskiq-whoisiq/info.json:624]: ok

**Pull request merged**